apisix/cli/ops.lua

```diff
@@ -265,7 +265,7 @@ local config_schema = {
         http = {
             type = "object",
             properties = {
-                lua_shared_dicts = {
+                custom_lua_shared_dict = {
```

Comment:
will this impact the users upgrade from old version of APISIX?

Reply:
lua_shared_dicts is still available now (only a warn message will generate).